CHANGELOG.md

```diff
@@ -9,10 +9,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
-### Fixed  
+### Fixed
 
 ### Changed
 
+## [2.1.1] - 2022-09-14
+
+### Fixed
+
+- Remove GOCART requirement for gFTL, gFTL-Shared, pFlogger and yaFyaml. These are requirements of MAPL. (#184)
```

Comment:
Is gFTL-Shared not required? It seems required

Reply:
Dang it. Didn't fix the changelog when I made the last change. Fixing now